pandas/core/dtypes/common.py

```diff
@@ -1892,9 +1892,6 @@ def _is_dtype_type(arr_or_dtype, condition):
     bool : if the condition is satisifed for the arr_or_dtype
     """
 
-    if arr_or_dtype is None:
```

Comment:
rather remove the other one

Reply:
Hmm, I just recommended removing this one so they're collected. Anyway, this is obviously not important, so I'll be bowing out.